apps/spruce/src/components/ScheduleTasksModal/index.tsx

```diff
@@ -70,6 +74,12 @@ export const ScheduleTasksModal: React.FC<ScheduleTasksModalProps> = ({
     dispatch({ type: "ingestData", taskData });
   }, [taskData]);
 
+  const numEstimatedActivatedGeneratedTasks =
+    getNumEstimatedActivatedGeneratedTasks(
```

Comment:
[nit] We tend to use "count" instead of "num" for naming
```suggestion
  const estimatedActivatedGeneratedTasksCount =
    getEstimatedActivatedGeneratedTasksCount(
```